bcda/web/api_test.go

```diff
@@ -196,6 +196,51 @@ func (s *APITestSuite) TestBulkCoverageRequestInvalidSinceFormatOnlyInvalidDate(
 	bulkCoverageRequestInvalidSinceFormatHelper("Group/all", since, s)
 }
 
+func (s *APITestSuite) TestBulkCoverageRequestInvalidSinceFormatOnlyJunkDataAfterDate() {
+	since := "2020-02-13T08:00:00.000-05:00dfghj"
+	bulkCoverageRequestInvalidSinceFormatHelper("Patient", since, s)
+	s.TearDownTest()
+	s.SetupTest()
+	bulkCoverageRequestInvalidSinceFormatHelper("Group/all", since, s)
+}
+
+func (s *APITestSuite) TestBulkCoverageRequestInvalidSinceFormatOnlyJunkDataBeforeDate() {
+	since := "erty2020-02-13T08:00:00.000-05:00"
+	bulkCoverageRequestInvalidSinceFormatHelper("Patient", since, s)
+	s.TearDownTest()
+	s.SetupTest()
+	bulkCoverageRequestInvalidSinceFormatHelper("Group/all", since, s)
+}
+
+func (s *APITestSuite) TestBulkCoverageRequestInvalidSinceFutureDate() {
+	futureDate := time.Now().AddDate(0, 0, 1).Format(time.RFC3339Nano)
+	bulkCoverageRequestInvalidSinceDateHelper("Patient", futureDate, s)
+	s.TearDownTest()
+	s.SetupTest()
+	bulkCoverageRequestInvalidSinceDateHelper("Group/all", futureDate, s)
+}
+
+func (s *APITestSuite) TestBulkCoverageRequestInvalidOutputFormatTextHTML() {
```

Comment:
Do we need three more test functions to test the positive cases where `_outputFormat` is set correctly?  (i.e., `ndjson`, `application/fhir+ndjson`, and `application/ndjson`)?